src/kdbindings/signal.h

```diff
@@ -242,6 +244,27 @@ class Signal
             return m_connections.insert({ slot });
         }
 
+        // Establish a deferred connection between signal and slot, where ConnectionEvaluator object
+        // used to queue all the connection to evaluate later. The returned
+        // value can be used to disconnect the function again.
+        Private::GenerationalIndex connectDeferred(const std::shared_ptr<ConnectionEvaluator> &evaluator, std::function<void(Args...)> const &slot)
+        {
+            auto weakEvaluator = std::weak_ptr<ConnectionEvaluator>(evaluator);
+
+            auto connection = [weakEvaluator, slot](Args... args) {
```

Comment:
```suggestion
            auto connection = [weakEvaluator = move(weakEvaluator), slot](Args... args) {
```
C++14 allows capturing by move, no need to copy the weak_ptr :blush: